zerver/lib/validator.py

```diff
@@ -588,6 +588,24 @@ def to_timezone_or_empty(var_name: str, s: str) -> str:
         return canonicalize_timezone(s)
 
 
+def to_bool(s: str) -> bool:
+    """
+    This function is essentially django.forms.BooleanField.to_python,
+    the normalization function Django uses to convert strings representing
+    boolean values passed by form requests.
+    """
+    # Explicitly check for the string 'False', which is what a hidden field
+    # will submit for False. Also check for '0', since this is what
+    # RadioSelect will provide. Because bool("True") == bool('1') == True,
+    # we don't need to handle that explicitly.
+    if isinstance(s, str) and s.lower() in ("false", "0"):
+        value = False
+    else:
+        value = bool(s)
+
+    return value
```

Comment:
Ideally, we want to avoid adding a new function like this. https://github.com/zulip/zulip/pull/19741/commits/f662b0834cb157e4ba161534ee88290a28fa55c3#r788252405 has some more discussion. I don't have a specific solution in mind, but I'm not sure we actually need a boolean parameter in this endpoint at all.